JavaScriptSPA/msal-0.9.0.min.js

```diff
@@ -0,0 +1,17 @@
+/*! msal v0.2.4 2019-04-13 */
```

Comment:
Good idea to share this sample as is to Graph for integration. I like the bundling.  +1.